### ISSUE TITLE
LocaleUtil bug crash on paper 1.16.5

### ISSUE THREAD

**Issue by Mrredstone5230** [TownyAdvanced/FlagWar#43]
### What steps will reproduce the problem?
1.Build the latest source of the plugin
2.Open a 1.16 server (JDK 16)

### What is the expected output?
1.Server boots up normally !

### What is your Towny version number?
0.97.0.0

### What is your FlagWar version number?
Well, I would say "latest", but in the pom it's 0.1.2

### What is your Spigot/Paper/Fork server version?
This server is running Paper version git-Paper-675 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### Please use a pastebin service to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly.
If you do skip this step then your ticket will most likely be deleted, and you will be asked to resubmit.**
1. Your full server startup from the `logs\latest.log` file : https://pastebin.com/PbHqMUAW
2. Your FlagWar config : (The default config with "translation: "en_US"")
3. Your log's error : https://pastebin.com/hDd3McQZ

### What is the estimated problem
Instead of loading the "en_US" ressource, it loads "en__US", which makes a massive error in the console. That is easily fixable with the code right below.

### Ways to repair:
In LocaleUtil.java at line 75, change the line for 'region = localeString.substring(localeString.indexOf("_") + 1);'

(This is my first request on github, so sorry if it's wrongly made)

**Comment by TheFlagCourier** [comment]
I noticed your log error still says `FlagWar v0.1.1` - so something is definitely off here. I've had no such issue using a fresh clone on JRE 11 (Amazon Corretto), nor on JRE 16 (OpenJDK).

Please ensure that you _do not_ have a Flagwar-0.1.1.jar in your  plugins folder, and that your local clone is up-to-date. The output of `git log` should start with "commit d1dba83a3a9dbfa2648bf222002965ad984299d7 (HEAD -> main, origin/main, origin/HEAD)" (for this point in time.)

If it does not, please do a `git fetch && git pull`, then rebuild with maven.

**Comment by Mrredstone5230** [comment]
Well, I don’t know what’s up with the plugin version, but it’s definitely custom builded by myself. The git version was also the latest. I’m also not sure what’s the jdk version. I know I use open-jdk 8, 11 and 16 on my machine, but I’ll be able to give more details on that later today...

**Comment by Mrredstone5230** [comment]
Sorry, I thought it was openjdk, but I have Oracle's Java 16 SE Runtime (For the server). But I have the latest repo version installed and sorry about the "FlagWar v0.1.1", it's because I originally did the test on FlagWar v0.1.1. I then saw that even on the latest version it doesn't work, so I just took the first log I had (which was the one with FlagWar v0.1.1). I will try to make a pull request soon (if I understand correctly how they work :/)

**Comment by TheFlagCourier** [comment]
Committed some debug messages into main to assist with this (https://github.com/TownyAdvanced/FlagWar/commit/63bd3768bda0f8c38ce58ceb2498ba7e2e881ee5). Re-based your changes (#44) on top, locally, and tested. Confirmed that it tries to set region as `_Region` rather than `Region`

It certainly fixes issues with simple translations like `en_GB`, but doesn't seem to address issues with _ResouceBundles_ containing variants, where regions may end up looking like `_Region_variant`, among other issues. (ie: `en_US_POSIX` / `en_US-POSIX`).

I've got a solution typed up that I'm testing out and may add it to your PR to keep it bundled together.
